engine.go

```diff
@@ -205,10 +205,13 @@ func (e *RTCEngine) Close() {
 		}
 
 		e.onCloseLock.Lock()
-		for _, onCloseHandler := range e.onClose {
+		onClose := e.onClose
+		e.onClose = []func(){}
```

Comment:
Is there a need to clear `onClose`. Guess it can be left as is right?

Reply:
Yeah it can be left as is as well, but also no harm in cleaning these up instead of waiting for the go gc, since these won't be used again